ci/deploy/pipeline.yml

```diff
@@ -39,6 +39,20 @@ steps:
       manual:
         permit_on_passed: true
 
+  - command: bin/ci-builder run stable bin/pyactivate -m ci.deploy.npm
+    timeout_in_minutes: 30
+    concurrency: 1
+    concurrency_group: deploy/npm
+    agents:
+      queue: linux-x86_64
+    build:
+      env:
+        BUILDKITE_BUILD_NUMBER: "$BUILDKITE_BUILD_NUMBER"
+        NPM_TOKEN: "$NPM_TOKEN"
```

Comment:
The environment variables should be passed through automatically?

```suggestion
```

Reply:
Ah! I was under the impression some weren't because we're explicitly passing the `BUILDKITE_TAG` to the SQL Logic and Nightly tests. Will remove.

Reply:
Ah, that's because those are trigger steps! When you trigger a different pipeline you have to explicitly plumb through the environment variables you want to retain. Normal command steps have access to all environment variables.